Formula/whalebrew.rb

```diff
@@ -19,7 +19,11 @@ class Whalebrew < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+
+    (bash_completion/"whalebrew").write Utils.safe_popen_read(bin/"whalebrew", "completion", "bash")
+    (zsh_completion/"_whalebrew").write Utils.safe_popen_read(bin/"whalebrew", "completion", "zsh")
+    (fish_completion/"whalebrew.fish").write Utils.safe_popen_read(bin/"whalebrew", "completion", "fish")
```

Comment:
```suggestion
    generate_completions_from_executable(bin/"whalebrew", "completion")
```

Reply:
The DSL is awesome, thanks so much for adding that!